heka/sandbox/encoders/http_edge_encoder.lua

```diff
@@ -68,24 +61,25 @@ local ns_config = {
 }
 
 function process_message()
-    -- Carry forward payload some incoming fields
```

Comment:
In general we initialize msg with all the fields we use:
1) to avoid additional allocation/table resizing on the first invocation of process_message
2) to document the message structure (this applies to Fields too although you have a bunch of dynamic ones here)